prow/clonerefs/parse_test.go

```diff
@@ -168,20 +168,20 @@ func TestParseRefs(t *testing.T) {
 	}
 }
 
-func TestParseAliases(t *testing.T) {
+func TestParseOverride(t *testing.T) {
```

Comment:
Seems like we ought to explicitly have the whole k8s.io to github.com/kubernetes thing in here as well as the `https://github.com/kubernetes/test-infra` to `git@github.com:kubernetes/test-infra.git` thing

Reply:
This mapping (`org/repo=value`) is used for both. What do you mean by this comment?

Reply:
I'm expecting to see a test that has an input of something like `k8s.io/test-infra` and validates an output of `git@github.com:fejta/test-infra.git`.

Does this already exist somewhere?

Reply:
This test ensures that the string `org,repo=value` is parsed as a mapping from `(org,repo)` to `value`. It's opaque to any specific k8s.io semantics or otherwise. I'm still lost as to what you want to have tested here. If a user provides a flag in the appropriate form, they can set the clone URI to `google.com` for all we care here.